### PR TITLE
add recursive option to folder API

### PR DESCRIPTION
**Pull request by daniel-lang**
This is a follow up to [PR #6545 for cakephp/cakephp](https://github.com/cakephp/cakephp/pull/6545) to add the option that was added in the PR.

I added the option to every language available but only added a comment to the english version since I don't speak the others.

### CODE REVIEW

**Comment by daniel-lang** [comment]
Since CakePHP 3.0.5 is out containing these changes, I think it is time to merge this to show the changes in the documentation.

**Comment by antograssiot** [comment]
@daniel-lang the sentence just before the code sample is still : 
"Recursively copy a directory".
Maybe it would be good to update to "Copy a directory (recursively by default)" or something similar ?

**Comment by daniel-lang** [comment]
Changed it. Also changed the sentence at the move() method.

File: en/core-utility-libraries/file-folder.rst [hunk 1]
```diff
@@ -287,7 +288,7 @@ Folder API
 
     :rtype: boolean
 
-    Recursive directory move.
+    Move a directory (recursively by default). The only parameter $options is the same as for ``copy()``
```
Comment (daniel-lang) [inline en/core-utility-libraries/file-folder.rst]: I couldn't figure out how to cross-refer to the copy method

**Pull request merged**